base/status/StatusStatistics.ts

```diff
@@ -0,0 +1,376 @@
+            cursor_y = StatusStatistics.CURSOR.EFFECT.Y;
+        }
+        this.update_highlight(highlight);
+        
+        let cursor_tween = {type: CursorManager.CursorTweens.POINT, variant: PointVariants.SHORT};
+        this.data.cursor_manager.move_to({x: cursor_x, y:cursor_y}, {animate: false, flip: true, tween_config: cursor_tween});
+    }
+
+    public on_change(){
+        this.select_option();
+        if(this.current_col===0){
+            let msgs = {line1: StatusStatistics.StatisticsMsgs[this.current_line].line1,
+                line2: StatusStatistics.StatisticsMsgs[this.current_line].line2};
+
+            if(this.current_line === Statistics.EXP){
+                let char = this.manager.selected_character;
+                let exp = char.exp_curve[char.level] - char.current_exp;
+                msgs.line2 = msgs.line2.replace("${EXP}", exp);
+            }
+            this.manager.update_description(msgs.line1, msgs.line2);
+        }
+        else{
+            if(this.manager.battle_effects_array.length === 0){
+                this.manager.update_description("Normal status.", "");
+            }
+            else{
+                let effect = this.manager.battle_effects_array[this.current_col-1];
+
+                let msgs = null;
+                if(effect.type === BattleEffectTypes.STATUS_CONDITION){
+                    let name = (effect.key as string).toUpperCase();
+                    msgs = {line1: StatusStatistics.BattleStatusMsgs[name].line1,
+                        line2: StatusStatistics.BattleStatusMsgs[name].line2};
+
+                    if(effect.key === temporary_status.DEATH_CURSE){
+                        let turns = (effect.properties.turns ? effect.properties.turns : 0);
+                        msgs.line1 = msgs.line1.replace("${N}", turns);
+                    } 
+                }
+                else if(effect.type === BattleEffectTypes.BUFF_DEBUFF){
+                    if(effect.key.includes("res") || effect.key.includes("pow")){
+                        let effect_name = "";
+                        if(effect.key.includes("res")) effect_name = "Resist";
+                        else if(effect.key.includes("pow")) effect_name = "Power";
+
+                        let venus = String((effect.properties.values[0] ? effect.properties.values[0] : 0));
+                        let mercury = String((effect.properties.values[1] ? effect.properties.values[1] : 0));
+                        let mars = String((effect.properties.values[2] ? effect.properties.values[2] : 0));
+                        let jupiter = String((effect.properties.values[3] ? effect.properties.values[3] : 0));
+
+                        let elems_to_show:{element:string, value:string}[] = [];
+                        msgs = {line1: "", line2: ""};
+
+                        if(parseInt(venus) !== 0) elems_to_show.push({element: elements.VENUS, value: venus});
```

Comment:
Why not removing line 185 (`let venus = String((effect.p...`) and then doing like so:
```javascript
const venus = effect.properties.values[0];
if(venus) elems_to_show.push({element: elements.VENUS, value: String(venus)});
```
Also, why using indexes for elements? Why not using the element name:
```javascript
const venus = effect.properties.values.venus;
...
```